tests/01-sanity.sh

```diff
@@ -29,4 +29,18 @@ stest 1 sudo false
 stest 0 which singularity
 stest 0 test -f "$SINGULARITY_sysconfdir/singularity/singularity.conf"
 
+# Check whether singularity binary is on a volume mounted with 'nosuid'
+# I guess one could just grep /etc/mtab, but...
+mount | grep $(df -h $SINGULARITY_PATH | grep -v File | awk '{print $1}') \
+    > $SINGULARITY_TESTDIR/singularity_fs_nosuid
+stest 1 grep "nosuid" $SINGULARITY_TESTDIR/singularity_fs_nosuid
+
+# Is the SINGULARITY_PATH going to be there when you sudo?
+sudo grep "^Defaults.*secure_path=.*$SINGULARITY_PATH" /etc/sudoers
+echo $? > $SINGULARITY_TESTDIR/singularity_secure_path
+stest 0 grep 0 $SINGULARITY_TESTDIR/singularity_secure_path
```

Comment:
What would happen if you just did something like?
```
stest 0 sudo singularity --version
```

Reply:
grepping for secure_path would fail for me as I don't have it set. So.. since the grep fails it returns 1, but running `sudo singularity` works fine.

```
$ sudo grep '^Defaults.*secure_path' /etc/sudoers
$ echo $?
1
$ which singularity
/usr/local/singularity/devel/bin/singularity
$ sudo singularity --version
2.4.1-development.g08dc27a
```